python/looker_sdk/sdk/api31/models.py

```diff
@@ -196,7 +242,7 @@ class ContentMetaGroupUser(model.Model):
     can: Optional[MutableMapping[str, bool]] = None
     id: Optional[str] = None
     content_metadata_id: Optional[str] = None
-    permission_type: Optional[str] = None
+    permission_type: Optional[models.PermissionType] = None
```

Comment:
need an additional tweak to python formatter: these type annotations need to be quoted ([lazy type evaluation](https://www.python.org/dev/peps/pep-0563/) is only 3.7+) and they're defined inside this same module so no need for the `models.` access:

```suggestion
    permission_type: Optional["PermissionType"] = None
```

in short, should be same as `ApiVersion` for example

Reply:
another option would be to leave out these python generated changes and merge it in. Then I can do the fixup to get the annotations correct as well as verify that the [de]serialization is working

Reply:
Let's hold off on merging this PR until Python is right, too.

Reply:
Got the quote in there, but forwardrefs for enum types may still need to be generated?